README.md

```diff
@@ -55,7 +52,7 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Original design idea by Cindy Shin in Behance.
+- Original design idea by <a href="https://www.behance.net/adagio07">Cindy Shin in Behance.</a>
 
 
 ## 📝 License
```

Comment:
- [OPTIONAL] the link to MIT file in your README is broken, kindly fix it by updating the path to `[MIT](./MIT.md) ` instead of  `[MIT](./LICENSE) `😉

![image](https://user-images.githubusercontent.com/45231276/192104945-204ce237-d30d-4883-b481-5f74c27923ac.png)

- [OPTIONAL] And Remember to customize your MIT.md file, adding your name and customize it 💯

![image](https://user-images.githubusercontent.com/45231276/192119189-7ffe29d8-7845-4ed2-854a-bfdf7abe6cb2.png)